logstash-plugins/usagetype-plugin/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
+##
+##############################################################################
+
+# Attempt to set APP_HOME
+# Resolve links: $0 may be a link
+PRG="$0"
```

Comment:
What is PRG?

Reply:
Good question! It looks like it takes the value of the first argument the the script. To write a logstash plugin, you need to copy the logstash plugin template that Elastic provides. The tutorial is here: https://www.elastic.co/guide/en/logstash/current/java-filter-plugin.html

I'll admit I haven't worked much with gradle before so I really don't know all the ins and outs of it. But gradle is the toolchain used to build the plugins, so there will be some gradle stuff in this PR. Not sure I can answer all the in-depth questions, though.

Reply:
Ah ok, I guess I wasn't sure what code was yours and what code was from somewhere else. I hadn't realized I was in the logstash-plugins folder